dist/css/bootstrap.css

```diff
@@ -11107,10 +11099,10 @@ a[data-toggle=tooltip] {
 }
 .popover.bottom-left {
   margin-top: 14px;
-  -webkit-transform-origin: -webkit-calc(100% - 29px) 0;
-      -ms-transform-origin:         calc(100% - 29px) 0;
-       -o-transform-origin:         calc(100% - 29px) 0;
-          transform-origin:         calc(100% - 29px) 0;
+  -webkit-transform-origin: calc(100% - 29px) 0;
+      -ms-transform-origin: calc(100% - 29px) 0;
+       -o-transform-origin: calc(100% - 29px) 0;
```

Comment:
Do we know if these will fair any better on the crab LESS compilation side? Or should these get removed too?

Reply:
hmm yeah for some reason I didn't consider other prefixes, I'll look into it

Reply:
actually, missed the fact that this isn't a calc prefix, so these should work fine. Seems Less just has an exception for `calc` which doesn't apply if there is a prefix.

Reply:
<img width="380" alt="Screenshot 2020-07-08 at 16 55 19" src="https://user-images.githubusercontent.com/43746750/86927292-d3458000-c13b-11ea-8a7c-84ec04dce4ea.png">

Reply:
Ah, right, totally missed the fact that it's the calc that has the prefix 😂